gems/pending/spec/middleware/events/middleware_event_monitor_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "middleware/events/middleware_event_monitor"
+
+describe MiddlewareEventMonitor do
+  subject do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
+    auth = AuthToken.new(:name => "test", :auth_key => "valid-token", :userid => "jdoe", :password => "password")
+    ems  = FactoryGirl.create(:ems_hawkular,
+                              :hostname        => 'localhost',
+                              :port            => 8080,
+                              :authentications => [auth])
+    WebMock.allow_net_connect!
+    described_class.new(ems)
+  end
+
+  context "#perf_catch_events" do
+    # VCR.eject_cassette
+    # VCR.turn_off!(ignore_cassettes: true)
+
+    VCR.configure do |c|
+      c.default_cassette_options = {
+        :match_requests_on => [:method, VCR.request_matchers.uri_without_params(:startTime, :endTime)]
+      }
+    end
+
+    it "will invoke a perf_catch_events" do
+      prefix = 'manageiq/providers/hawkular/'
+      VCR.use_cassette(prefix + described_class.name.underscore.to_s) do # , :record => :new_episodes) do
```

Comment:
I had it that way but changed it because it generates a cassette at the top level cassette directory (and no other cassettes are there).  I thought perhaps it was better to get it under the hawkular provider cassettes.  But I will change it back for consistency.